File: 2000 98 & ME/QuickStart/howto/samples/Services/Installers/customaction/vb/projectinstaller.vb
Imports System
Imports System.Configuration.Install
Imports System.ComponentModel

public class <RunInstaller(true)> ProjectInstaller: Inherits Installer

	private customInstaller As CustomInstaller

	public sub New()
		MyBase.New()
		customInstaller = new CustomInstaller()
		Installers.Add(customInstaller)
	end sub
end class

File: 2000 98 & ME/QuickStart/howto/samples/Services/Installers/customaction/projectinstaller.cs
using System;
using System.Configuration.Install;
using System.ComponentModel;

[RunInstaller(true)]
public class ProjectInstaller: Installer{

	private CustomInstaller customInstaller;

	public ProjectInstaller(){

		customInstaller = new CustomInstaller();
		Installers.Add(customInstaller);
	}
}
